sourcecode/hub/resources/css/theme.scss

```diff
@@ -13,4 +13,35 @@
     .navbar-bottom {
         background-color: $green-200;
     }
+
+    p, h1 {
+        font-family: Helvetica Neue;
+    }
+
+    .btn-custom {
+        background: linear-gradient(0deg, #FFFFFF, #FFFFFF);
+        border: 1px solid #CED4DA;
+    }
+
+    .no-content-found-title {
```

Comment:
With the exception of the `height: 50vh` rule, all of this CSS can be replaced with utility classes from Bootstrap. The fonts, font sizes, line heights, etc. should all be the default from Bootstrap, or use Bootstrap's variables, so we don't end up in a situation where we have 100 custom elements and 5 billion lines of CSS for each of them that need to be updated for every theme we end up making.